integration-tests/helpers/index.js

```diff
@@ -269,17 +270,22 @@ async function createSandbox (dependencies = [], isGitRepo = false,
     // ... run the tests in the current directory.
     return { folder: path.join(process.cwd(), 'integration-tests'), remove: async () => {} }
   }
-  const folder = path.join(os.tmpdir(), id().toString())
-  const out = path.join(folder, `dd-trace-${version}.tgz`)
-  const allDependencies = [`file:${out}`].concat(cappedDependencies)
+  const folder = path.join(sandboxRoot, id().toString())
+  const out = path.join(sandboxRoot, 'dd-trace.tgz')
+  const deps = cappedDependencies.concat(`file:${out}`)
 
-  await fs.mkdir(folder)
-  const preferOfflineFlag = process.env.OFFLINE === '1' || process.env.OFFLINE === 'true' ? ' --prefer-offline' : ''
-  const addCommand = `yarn add ${allDependencies.join(' ')} --ignore-engines${preferOfflineFlag}`
+  await fs.mkdir(folder, { recursive: true })
   const addOptions = { cwd: folder, env: restOfEnv }
```

Comment:
Tiny nit: This lines confuses me 😅 I know this nit is unrelated to this PR, but if you're changing other things, feel free to either just inline this where it's used below, or at least move the declaration of the constant down above where it's used. For longer than I'd like to admit I thought it was used in the `bun pm pack` command 🫣 Feel free to move the `addFlags` declaration as well 😊

But feel free to ignore this as it's just a cleanup of unrelated code.